modules/bitgo/test/v2/unit/internal/tssUtils/ecdsa.ts

```diff
@@ -545,7 +562,7 @@ describe('TSS Ecdsa Utils:', async function () {
         to: SignatureShareType.BITGO,
         share: userSShare.R + userSShare.s + userSShare.y + userOmicronAndDeltaShare.dShare.delta + userOmicronAndDeltaShare.dShare.Gamma,
       };
-      const getBitGoSShare = MPC.sign(signablePayload, getBitgoOShareAndDShares.oShare, userOmicronAndDeltaShare.dShare);
```

Comment:
This previously didn't actually work since it was combining sha256 and keccack256 hashes